src/main/java/mybatis/com/mybatis/Dto/SubjectCreationDto.java

```diff
@@ -1,2 +1,11 @@
-package mybatis.com.mybatis.Dto;public class SubjectCreationDto {
+package mybatis.com.mybatis.Dto;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import lombok.Data;
+import mybatis.com.mybatis.Entity.TeacherEntity;
+
+@Data
+public class SubjectCreationDto {
+    private String name;
+    private TeacherCreationDto TeacherCreationDto;
```

Comment:
Remove this and just accept the Long teacherId

Reply:
done